concepticondata/conceptlists/Izura-2005-499.tsv

```diff
@@ -0,0 +1,500 @@
+Izura-2005-499-256	256	Mesita	bedside table			furniture	6.36	5.48	5.04	6.84	42.025	2	3
+Izura-2005-499-257	257	Microondas	microwaves			furniture	2.64	5.32	11.42	7	8.237	3	4
+Izura-2005-499-258	258	Minicadena	mini hi-fi			furniture	2.28	5.44	10.88	6.96	1.89	0	5
+Izura-2005-499-259	259	Mural	mural			furniture	2	2.12	9	5.04	0.641	3	2
+Izura-2005-499-260	260	Nevera	fridge			furniture	2.72	5.96	6.52	6.96	6.282	12	3
+Izura-2005-499-261	261	Nido	truckle bed    			furniture	3.28	1.88	6.92	6	0.9	13	2
+Izura-2005-499-262	262	Ordenador	computer	COMPUTER	3086	furniture	2.24	6.44	10.75	6.96	4.671	67	4
+Izura-2005-499-263	263	Papelera	litter			furniture	2.84	6	6.24	7	2.398	4	4
+Izura-2005-499-264	264	Paragüero	umbrella stand			furniture	3.04	3.28	7.6	6.76	1.694	1	4
+Izura-2005-499-265	265	Percha	hanger			furniture	2.76	5.28	6.4	6.96	4.559	7	2
+Izura-2005-499-266	266	Perchero	clothes stand			furniture	3.84	3.88	7.12	6.68	5.286	4	3
+Izura-2005-499-267	267	Piano	piano			furniture	2.2	3.28	6.04	7	1.858	31	3
+Izura-2005-499-268	268	Portalibros	book shelf   			furniture	2.84	2.28	11.15	5.2	0.641	0	4
+Izura-2005-499-269	269	Posaplatos	place mat			furniture	2.04	2.36	11.64	5.36	0.679	0	4
+Izura-2005-499-270	270	Puerta	door	DOOR	1567	furniture	3.48	6.56	4.24	7	4.876	301	2
+Izura-2005-499-271	271	Puf	pouffe			furniture	3	2.48	10.86	5.08	3.812	0	1
+Izura-2005-499-272	272	Pupitre	school desk			furniture	5.44	3.68	6	6.76	1.752	9	3
+Izura-2005-499-273	273	Radio	radio	RADIO	1315	furniture	2.24	5.76	5.68	6.96	1.787	75	2
+Izura-2005-499-274	274	Recibidor	hall     			furniture	3.88	3.16	8.72	5.96	2.872	7	4
+Izura-2005-499-275	275	Reloj	clock	CLOCK	1323	furniture	2.52	6.24	4.92	7	2.519	43	2
+Izura-2005-499-276	276	Repisa	shelf	SHELF	745	furniture	3.6	4.04	7.84	6.64	1	3	3
+Izura-2005-499-277	277	Retrete	loo			furniture	1.8	3.36	8.64	7	1.261	14	3
+Izura-2005-499-278	278	Revistero	magazine rack			furniture	3.44	2.8	10.25	6.2	0.914	0	4
+Izura-2005-499-279	279	Rinconera	cornerboard			furniture	4.84	2.44	11.58	5.76	1.99	0	4
+Izura-2005-499-280	280	Ropero	wardrobe	WARDROBE	3106	furniture	6.08	3.32	9.84	6.48	2.893	6	3
+Izura-2005-499-281	281	Secreter	writing desk			furniture	4.44	1.36	16.89	4.12	1.215	0	3
+Izura-2005-499-282	282	Silla	chair	CHAIR	1476	furniture	6.8	6.72	4.64	7	78.805	58	2
+Izura-2005-499-283	283	Sillón	armchair			furniture	6.76	5.84	5	7	61.626	31	2
+Izura-2005-499-284	284	Sinfonier	chest of drawers			furniture	4.2	1.64	13.69	3.24	3.479	0	3
+Izura-2005-499-285	285	Sofá	sofa	SOFA	3099	furniture	6.8	6.36	6.24	7	56.795	4	2
+Izura-2005-499-286	286	Taburete	stool	STOOL	2058	furniture	5.72	4.48	6.08	6.96	18.16	2	4
+Izura-2005-499-287	287	Taquillón	traditional Spanish table			furniture	4.84	2.32	9.95	5.52	2.278	0	3
+Izura-2005-499-288	288	Televisión	television	TELEVISION	1788	furniture	4	6.24	4.35	6.8	14.852	85	4
+Izura-2005-499-289	289	Televisor	television			furniture	2.88	5.76	6.52	6.96	2.819	25	4
```

Comment:
if they have two separate elicitation glosses, you are allowed to link to the same concept set. These mergers result from synonymy in the concept list.

Reply:
Okay, that's good to know as this is also relevant for another list I'm preparing.